modules/bm_apps/Makefile.am

```diff
@@ -5,12 +5,16 @@ AM_CPPFLAGS += \
 -I$(srcdir)/include \
 -I$(top_builddir)/thrift_src/gen-cpp
 
+if P4THRIFT
+  AM_CPPFLAGS += -DP4THRIFT $(THRIFT_INC)
```

Comment:
Right I noticed that. Working on the style check now but it looks like it's failing on files I haven't changed.